packages/sdk/src/types/Validations.ts

```diff
@@ -45,11 +48,19 @@ export interface ValidationsValidateVoucherResponse {
 	metadata?: Record<string, any>
 	code?: string
 	valid?: boolean
-	discount: DiscountAmount | DiscountUnit | DiscountPercent
+	discount?: DiscountAmount | DiscountUnit | DiscountPercent
+	gift?: {
+		amount: number
+		balance: number
+	}
+	loyalty?: {
+		points_cost: number
+	}
 	order: {
```

Comment:
This is werid, so here we have everything with `?` but in the `ClientSideValidateResponse` we have everything without `?`. It should be the same in both, please check API and decide if all fields should be marked with `?` or not